test/core/card.spec.js

```diff
@@ -0,0 +1,22 @@
+const expect = require('./../helper').expect;
+
+const Card = require('./../../src/core/card');
+
+describe('Card', () => {
+  beforeEach(() => {
+    this.card = new Card(2, 'spades');
+  });
+
+  describe('#constructor', () => {
+    it('initializes a new card', () => {
+      expect(this.card.value).to.equal(2);
+      expect(this.card.seed).to.equal('spades');
```

Comment:
Also, when you're using constants in testings (like `2` or `spades`), make sure that you assign it to a variable that you always refer to: if I want to change `2` to `3` I should do it only in one place